automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -33,6 +33,13 @@ display_help() {
     echo
 }
 
+get_abs_filename() {
+  # $1 : relative filename
```

Comment:
Nit: IMO this comment isn't necessary since you're putting it into a variable named `relative_filename` on the next line

Reply:
Good call. Done!